it/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -0,0 +1,144 @@
+package controllers
+
+import models.{CheckMode, LitresInBands}
+import org.jsoup.Jsoup
+import org.scalatest.matchers.must.Matchers.{convertToAnyMustWrapper, include}
+import pages.{ContractPackingPage, HowManyContractPackingPage, HowManyImportsPage, HowManyOperatePackagingSitesPage, ImportsPage, OperatePackagingSitesPage}
+import play.api.http.HeaderNames
+import play.api.http.Status.OK
+import play.api.i18n.Messages
+import play.api.libs.json.Json
+import play.api.test.WsTestClient
+
+class CheckYourAnswersControllerISpec extends ControllerITTestHelper {
+
+  val route = "/check-your-answers"
+  "GET " + routes.CheckYourAnswersController.onPageLoad().url - {
+    "when the userAnswers contains no data" - {
+      "should render the page" in {
+        given
+          .commonPrecondition
+
+        setAnswers(emptyUserAnswers)
+
+        WsTestClient.withClient { client =>
+          val result = createClientRequestGet(client, baseUrl + route)
+
+          whenReady(result) { res =>
+            res.status mustBe OK
+            val page = Jsoup.parse(res.body)
+            page.title must include(Messages("checkYourAnswers.title"))
+            page.getElementsByClass("govuk-summary-list").size() mustBe 0
+          }
+        }
+      }
+    }
+    "when the userAnswers contains data" - {
```

Comment:
There is only tests for when the user has selected "yes" and then been presented with the litres in bands but none for when "no" was selected.